requirements.txt

```
@@ -20,7 +20,7 @@ requests==2.28.1
     # via
     #   alpino-query
     #   mwe-query (setup.py)
-sastadev==0.0.3
+-e git+https://github.com/UUDigitalHumanitieslab/sastadev@feature/packaging#egg=sastadev
```

```suggestion
sastadev==0.1.0
```
after the changes in sastadev are merged and uploaded to pypi